src/index.html

```diff
@@ -1,13 +1,431 @@
+            <p class="services__paragraph">We are sure that first-rate job is possible only if all three components are united.
+            </p>
+          </article>
+
+          <article class="process">
+            <div class="process__card process__card--desktop">
+              <span class="process__number">001</span>
+              <ul class="process__items">
+                <li>Brand Development</li>
+                <li>Copywriting</li>
+                <li>Logo & Webite Design</li>
+                <li>Packaging</li>
+              </ul>
+
+              <a href="#our-expertise" class="process__button">
+                <div class="process__arrow"></div>
+                <p class="process__text">Learn more</p>
+              </a>
+            </div>
+
+            <div class="process__card process__card--transform">
+              <span class="process__number">002</span>
+              <ul class="process__items">
+                <li>Content Production</li>
+                <li>Graphic Design</li>
+                <li>Video Production</li>
+                <li>Post Production</li>
+              </ul>
+
+              <a href="#our-expertise" class="process__button">
+                <div class="process__arrow"></div>
+                <p class="process__text">Learn more</p>
+              </a>
+            </div>
+
+            <div class="process__card process__card--desktop">
+              <span class="process__number">003</span>
+              <ul class="process__items">
+                <li>Marketing Strategy</li>
+                <li>Email Marketing</li>
+                <li>Paid Advertising</li>
+                <li>Blog Content & SEO</li>
+              </ul>
+
+              <a href="#our-expertise" class="process__button">
+                <div class="process__arrow"></div>
+                <p class="process__text">Learn more</p>
+              </a>
+            </div>
+
+            <div class="process__card process__card--transform">
+              <span class="process__number">004</span>
+              <ul class="process__items">
+                <li>Digital Communications</li>
+                <li>Influencer Marketing</li>
+                <li>Product Placements</li>
+                <li>Strategic Partnerships</li>
+              </ul>
+
+              <a href="#our-expertise" class="process__button">
+                <div class="process__arrow"></div>
+                <p class="process__text">Learn more</p>
+              </a>
+            </div>
+          </article>
+        </section>
+
+        <section class="testimonials" id="testimonials">
+          <h2 class="testimonials__title">
+            <p class="testimonials__title-text">What</p>
+            <p class="testimonials__title-text">people say</p>
+          </h2>
+
+
+          <article class="testimonials__card-container">
+            <div class="testimonials__card">
+              <img
+              src="images/Testimonials/Tal.Gilad.png"
```

Comment:
please refactor the path to a folder to math relative path, it's more secure `src="./images/Testimonials/Tal.Gilad.png"`, please fix that for whole file